beacon_node/http_api/src/lib.rs

```diff
@@ -3098,6 +3102,50 @@ pub fn serve<T: BeaconChainTypes>(
                 })
             },
         );
+    // TODO(zkproofs): I think we want to return a block_with_witness
```

Comment:
Here block refers to EL block -- okay given it merges with verkle stateless abstraction